folium/plugins/locate_control.py

```diff
@@ -0,0 +1,61 @@
+"""Add Locate control to folium Map.
+
+Based on leaflet plugin: https://github.com/domoritz/leaflet-locatecontrol
+"""
+
+from branca.element import CssLink, Figure, JavascriptLink, MacroElement
+
+from jinja2 import Template
+
+
+class LocateControl(MacroElement):
+    """Control plugin to geolocate the user.
+
+    Parameters
+    ----------
+    options: dict, optional
+        For possible options, see https://github.com/domoritz/leaflet-locatecontrol
+
+    Examples
+    --------
+    >>> import folium
```

Comment:
These examples are quite verbose. You can do without the folium importing, creating a map, etc.